tests/acceptance/features/apiActivities/activitiesByFileId.feature

```diff
@@ -0,0 +1,500 @@
+Feature: check activities
+  As a user
+  I want to check who made which changes to files using file-id
+  So that I can track modifications
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And using spaces DAV path
+
+
+  Scenario: check add activity for a file using file-id
+    Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile.txt"
+    And we save it into "FILEID"
+    And user "Alice" has created folder "/newFolder"
+    And user "Alice" has copied file "/textfile.txt" into "newFolder" inside space "Personal" using file-id "<<FILEID>>"
+    When user "Alice" lists the activities of file "newFolder" from space "Personal" using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["value"],
+        "properties": {
+          "value": {
+            "type": "array",
+            "minItems": 2,
+            "maxItems": 2,
+            "uniqueItems": true,
+            "items": {
+              "oneOf": [
+                {
+                  "type": "object",
+                  "required": ["id","template","times"],
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "pattern": "^%user_id_pattern%$"
+                    },
+                    "template": {
```

Comment:
do minimum check of this activity entry, we are more concern with second activity in this scenario

Reply:
@saw-jan I dont know why but for some reasons this test fails if i only check the `message` in first activity. That's why I checked the whole activity including the variables in first activity(which made it longer).

Reply:
may be because there are same messages in both of the activity❓

Reply:
> may be because there are same messages in both of the activity❓

yes, that is likely. you add some unique checks there